styles/global.scss

```diff
@@ -26,6 +26,10 @@ body {
       }
     }
   }
+  
+  a.dropdown-item{
+    padding-right: 1px;
+  }
```

Comment:
Why make it global?

Reply:
The default value of padding-right for `a.dropdown-item` is `3rem`. I thought this was way too large a value for right padding. That's why I added it to the `global.scss`.

Reply:
Hey, also can we remove the src text - `KodaDot NFT minted multimedia`? This displays before an image is loaded and doesn't look nice.

Reply:
> The default value of padding-right for `a.dropdown-item` is `3rem`. I thought this was way too large a value for right padding. That's why I added it to the `global.scss`.

we're using dropdown-item at various place, I don't know if this global modification can break a piece of design.


> Hey, also can we remove the src text - `KodaDot NFT minted multimedia`? This displays before an image is loaded and doesn't look nice.

I think this come from `<BasicImage />` component it's an alternative text if the image won't load we need to keep it for seo/accessibility purposes. 
It would be perfect if you could hide it only on this component (SearchBar), you could try out some hacks (font-size: 0; or opacity: 0;)

Reply:
> we're using dropdown-item at various place, I don't know if this global modification can break a piece of design.

I have now moved the global part to `SearchBar.vue`

> I think this come from `<BasicImage />` component it's an alternative text if the image won't load we need to keep it for seo/accessibility purposes. It would be perfect if you could hide it only on this component (SearchBar), you could try out some hacks (font-size: 0; or opacity: 0;)

I tried the `font-size:0` and it seems to have done the trick.